static/js/settings_org.js

```diff
@@ -1088,6 +1091,15 @@ export function build_page() {
             "id_realm_message_content_edit_limit_minutes",
             msg_edit_limit_dropdown_value === "custom_limit",
         );
+        if (msg_edit_limit_dropdown_value === "custom_limit") {
+            const val = get_property_value("realm_message_content_edit_limit_minutes");
+            if (Number.parseInt(val, 10) === 0) {
+                $("#id_realm_message_content_edit_limit_minutes")
+                    .closest(".org-subsection-parent")
+                    .find(".subsection-changes-save button")
+                    .prop("disabled", true);
```

Comment:
This is added because we currently show `0` in the input, when setting is cahnged from `anytime` to `custom`. One option can be we can just leave the box empty in that case.